# origin: Debian (herbert)
# cset: n/a
# inclusion: submitted (2004-09-18)
# description: use msgbox (with OK button) instead of infobox + sleep
# revision date: 2004-09-05

diff -urN kernel-source-2.4.26/scripts/Menuconfig kernel-source-2.4.26-1/scripts/Menuconfig
--- kernel-source-2.4.26/scripts/Menuconfig	2002-08-03 10:39:46.000000000 +1000
+++ kernel-source-2.4.26-1/scripts/Menuconfig	2003-02-22 14:08:24.000000000 +1100
@@ -490,10 +490,9 @@
 		case "$2" in
 		y)	echo -en "\007"
 			${DIALOG} --backtitle "$backtitle" \
-				  --infobox "\
+				  --msgbox "\
 This feature depends on another which has been configured as a module.  \
-As a result, this feature will be built as a module." 4 70
-			sleep 5
+As a result, this feature will be built as a module." 6 70
 			eval $1=m ;;
 		m)	eval $1=m ;;
 		c)	eval x=\$$1
@@ -557,8 +556,7 @@
 				eval $2=\"$3\"
 				echo -en "\007"
 				${DIALOG} --backtitle "$backtitle" \
-					--infobox "You have made an invalid entry." 3 43
-				sleep 2
+					--msgbox "You have made an invalid entry." 5 43
 			fi
 
 			break
@@ -590,8 +588,7 @@
 				eval $2=\"$3\"
 				echo -en "\007"
 				${DIALOG} --backtitle "$backtitle" \
-					--infobox "You have made an invalid entry." 3 43
-				sleep 2
+					--msgbox "You have made an invalid entry." 5 43
 			fi
 
 			break
@@ -957,8 +954,7 @@
 			else
 				echo -ne "\007"
 				$DIALOG	--backtitle "$backtitle" \
-					--infobox "File does not exist!"  3 38
-				sleep 2
+					--msgbox "File does not exist!"  5 38
 			fi
 		else
 			cat <<EOM >help.out
@@ -1021,8 +1017,7 @@
 			else
 				echo -ne "\007"
 				$DIALOG	--backtitle "$backtitle" \
-					--infobox "Can't create file!  Probably a nonexistent directory." 3 60
-				sleep 2
+					--msgbox "Can't create file!  Probably a nonexistent directory." 5 60
 			fi
 		else
 			cat <<EOM >help.out

